src/nemo_evaluator/solvers/harbor.py

```
@@ -917,6 +921,73 @@ async def _wait_for_agent(
 
         return timed_out, agent_error
 
+    async def _inject_skill(self, sandbox: "Sandbox", prompt: str) -> str:
```

Ruff (0.15.14)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)